eZ/Publish/Core/IO/Exception/InvalidBinaryFileIdException.php

```diff
@@ -8,12 +8,22 @@
  */
 namespace eZ\Publish\Core\IO\Exception;
 
+use Exception;
 use eZ\Publish\Core\Base\Exceptions\InvalidArgumentValue;
 
 class InvalidBinaryFileIdException extends InvalidArgumentValue
```

Comment:
It would mostly move your changes around. The places where you have added the message must be updated to use this new exception.

Or am I missing something ?